noxfile.py

```diff
@@ -112,6 +112,8 @@ def test(session, pandas_version: str):
         "python",
         "-m",
         "pytest",
+        "-ra",
```

Comment:
This reports all test except passes, which is useful to see why a specific would have been skipped.